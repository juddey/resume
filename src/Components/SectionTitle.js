import React from 'react'
import { Typography } from '@material-ui/core'

const SectionTitle = props => (
  <Typography variant='title'>{props.title}</Typography>
)

export default SectionTitle
