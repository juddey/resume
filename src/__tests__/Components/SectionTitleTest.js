import React from 'react'
import { createComponentWithIntl } from '../../../src/Support/Utils'
import SectionTitle from '../../Components/SectionTitle'

const div = document.createElement('div')

test('renders to snapshot', () => {
  const tree = createComponentWithIntl(
    <SectionTitle title='Section Title Component' />,
    div
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
