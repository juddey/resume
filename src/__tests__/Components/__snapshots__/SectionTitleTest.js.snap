// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to snapshot 1`] = `
<h3
  className="MuiTypography-root-1 MuiTypography-subheading-8"
  style={
    Object {
      "color": undefined,
      "textTransform": "uppercase",
    }
  }
>
  Section Title Component
</h3>
`;
